docs/BUILD.bazel

```diff
@@ -677,6 +677,7 @@ daml_test(
 daml_test(
```

Comment:
Last time I checked, `daml_test` runs all _scenarios_, not scripts. I'd like to keep the source files tested. Any chance we could extedn `daml_test` to also test scripts, or to have a separate `daml_test_scripts` rule?

Reply:
It does run the scripts!